### PR TITLE
fix: img should not overflow its container

### PR DESCRIPTION
**Pull request by AlanCutFlim**
### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link
N/A

### 💡 Background and solution
| before | after |
| --- | --- |
| image | image |

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |     Fix some img on website overflow its container    |
| 🇨🇳 Chinese |    修复网站部分图片溢出其容器的问题       |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: site/theme/static/markdown.less [hunk 1]
```diff
@@ -10,6 +10,7 @@
 
 .markdown img {
   max-width: ~'calc(100% - 32px)';
+  max-height: 100%;
```
Comment (yoyo837) [inline site/theme/static/markdown.less:13]: 容器加overflow如何？

Comment (AlanCutFlim) [reply site/theme/static/markdown.less:13]: > 容器加overflow如何？

容器加 overflow，那不是要么容器出现滚动条，要么 img 被裁切吗，应该都不符合吧

Comment (duuliy) [reply site/theme/static/markdown.less:13]: 请问下argos 是如何通过得吗

Comment (yoyo837) [reply site/theme/static/markdown.less:13]: 人工review判断修改是预期的然后Approve它

**Pull request merged**